.eslintrc.json

```diff
@@ -9,7 +9,10 @@
     "jest/globals": true,
     "node": true
   },
-  "rules": { "no-console": "error" },
+  "rules": {
+    "no-console": "error",
+    "no-unused-vars": ["error", { "argsIgnorePattern": "^_*" }]
```

Comment:
why have you implemented this? Is it outlined in coding standards? Are there any instances in the code where we use unused variables intentionally?

Reply:
that is part of a change that im not ready to commit, its.. complicated to explain without code so i'll just remove the change